examples/2D-navigation/css/presentation.css

```diff
@@ -247,20 +237,6 @@ body.impress-on-overview #crisps-image {
 
 
 /******************* PLUGINS *************************************************************/
-/*
-  This version of impress.js supports plugins, and in particular, a UI toolbar
-  plugin that allows easy navigation between steps and autoplay.
-*/
-.impress-enabled div#impress-toolbar {
-    position: fixed;
-    right: 1px;
-    bottom: 1px;
-    opacity: 0.6;
-    z-index: 10;
-}
-.impress-enabled div#impress-toolbar > span {
-    margin-right: 10px;
-}
 
```

Comment:
All of the below can be moved/removed too. Same for the other files.

Reply:
Should be done now.